### PR TITLE
Firefox 106 MR: add home promo, update /new features [fix #12225]

### LINKED ISSUE #12225: Firefox Major Release website updates
## Description

Firefox major release is coming, and there will be two changes to the website:
- top hero/split unit on homepage (multiple languages)
- changes to "latest features" on /new (english)
- adding "latest feature" to /new (other languages.

Details and assets should be delivered on Oct 12, and this card will be updated to include assets, designs, and success criteria. 

---

#### Success Criteria

- [ ] Send users to correct CTA, depending if they are mobile/desktop firefox/non-firefox. [Details](https://docs.google.com/presentation/d/1UeF90tYSkiQuWLKaIykinhGYzPhNSQ4PTxZmjqICzNE/edit#slide=id.g1584fcceb54_1_132)
- [ ] Event tracking in place

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary

Adds a home page promo for MR (Firefox 106) in English, French and German. Controlled by the switch `homepage-firefox-mr106-promo`

Updates the latest features on /firefox/new in English, French and German. Controlled by the switch `download-firefox-mr106-promo`

## Issue / Bugzilla link
#12225 

## Testing
Demo server URL: https://www-demo4.allizom.org

Home EN: http://localhost:8000/en-US/
Home DE: http://localhost:8000/de/
Home FR: http://localhost:8000/fr/
/new EN: http://localhost:8000/en-US/firefox/new/
/new DE: http://localhost:8000/de/firefox/new/
/new FR: http://localhost:8000/fr/firefox/new/

The home page promo has two variations of copy and image, but three different CTAs. In different combinations that makes four distinct versions of the promo:

- [ ] <del>In desktop Firefox show "Make Firefox yours" and CTA is for Firefox mobile.</del>
- [ ] <del>In desktop non-Firefox show "Make Firefox yours" and CTA is to Download Firefox.</del>
- [ ] <del>n mobile Firefox show "Browsing has never been so seamless" and CTA is to make Firefox default.</del>
- [ ] <del>In mobile non-Firefox show "Browsing has never been so seamless" and CTA is for Firefox mobile.</del>

The code itself isn't too complex but all the languages and variations gets tricky so it could use some methodical cross-platform testing to make sure I got everything right.

Also note I didn't add any detection for whether Firefox is already default, it just links to the set-to-default landing page rather than the thanks page (since the CTA targets mobile anyway).

### CODE REVIEW

File: media/css/mozorg/home/home-firefox-mr106-promo.scss [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        background-color: $color-violet-60;
+    }
+
+    // Override Protocol spacing
+    &.mzp-t-split-nospace {
+        padding-top: 0;
+        padding-bottom: 0;
+    }
+
+    // Override Protocol wordmark
```
Comment (alexgibson) [inline media/css/mozorg/home/home-firefox-mr106-promo.scss:22]: Side related - we should probably file a protocol issue to update the Firefox word-mark if we're now dropping "browser"?

File: media/css/mozorg/home/home-firefox-mr106-promo.scss [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            margin-right: 0;
+        }
+    }
+
+    // Override Protocol button
+    .mzp-c-button.mzp-t-product {
+        background-color: $color-white;
+        border-color: $color-white;
+        color: $color-black;
+
```
Comment (alexgibson) [inline media/css/mozorg/home/home-firefox-mr106-promo.scss:41]: Suggestion (non-blocking): should button copy be centre aligned?

image

**Review by alexgibson — APPROVED** [review]
Looks great and seems to show me the right things on both desktop and mobile (I tested iOS but also spoofed our platform logic)

r+

File: media/css/mozorg/home/home-firefox-mr106-promo.scss (hunk 1 above)
Comment (craigcook) [reply media/css/mozorg/home/home-firefox-mr106-promo.scss:22]: https://github.com/mozilla/protocol-assets/issues/86

File: media/css/mozorg/home/home-firefox-mr106-promo.scss (hunk 2 above)
Comment (craigcook) [reply media/css/mozorg/home/home-firefox-mr106-promo.scss:41]: This seems to be an existing issue with the button styling in Protocol. We don't explicitly declare `text-align` and when a button is just a link it inherit from its parent. `button` elements have `text-align: center` in the browser styles but links don't. I'll add it here but it's something that should really be fixed upstream in Protocol. I'll file an issue there.

**Review by alexgibson — APPROVED** [review]
Tested the new logic and looks good, r++

**Pull request merged**